app/nation/app/Home-Urdu.css

```diff
@@ -9,7 +9,8 @@ h1,h2,h3,h4,h5,h6{
 }
 #dashboard
 {
-    height: 471px;
+  /*  height: 471px;*/
```

Comment:
This commented code was intentionally not removed from here because of future MUI work to be done on this page.